mycroft/util/log.py

```diff
@@ -86,7 +86,8 @@ def init(cls):
         cls.handler = logging.StreamHandler(sys.stdout)
         cls.handler.setFormatter(formatter)
 
-        config = mycroft.configuration.Configuration.get(remote=False)
+        config = mycroft.configuration.Configuration.get(cache=False,
```

Comment:
Should this be done in Configuration class directly? If remote make cache always false

Reply:
Not sure. For small stand-alone scripts I can see the use of caching but still excluding the remote.

Reply:
do any standalone scripts really use Configuration? as in, under what circumstance does a standalone script want to import the whole mycroft-core package? If those are rare and mainly internal scripts (like send bus message) can't that be accounted for a different way?

just thinking out loud, don't feel like this is a blocker in any way, i think the PR is good for merge the way it is :)

Reply:
Those are totally valid thoughts...

Another approach would be to cache the different configurations separately and use the cache...When investigating it looked like that at startup there are 4 or 5 uncached calls to the `.get()` before it's cached just to avoid the remote config.

So when calling with remote it would get a cache including the remote and when calling without it would get the cache that was generated with only locals

Reply:
Ake have you got examples where you think the cache without remote would be useful? Or is it more theoretical?

Having a single config seems conceptually simpler to me. But then the complexity already exists and this is really just mapping against the somewhat commonly used remote flag, so I can see the benefit of the two caches.